src/MyHealth/OverTime.tsx

```diff
@@ -144,27 +146,76 @@ const OverTime: FunctionComponent = () => {
   const { t } = useTranslation()
   const { dailyLogData } = useSymptomLogContext()
 
+  enum ViewSelection {
+    LIST,
+    CALENDAR,
+  }
+
+  const [viewSelection, setViewSelection] = useState(ViewSelection.LIST)
+
   const noSymptomHistory = dailyLogData.length === 0
 
+  const last14DaysStyle =
+    viewSelection === ViewSelection.LIST ? null : style.hidden
+
+  const historyStyle =
+    viewSelection === ViewSelection.CALENDAR ? null : style.hidden
+
   return (
-    <ScrollView
-      style={style.container}
-      contentContainerStyle={style.contentContainer}
-      alwaysBounceVertical={false}
-    >
-      {noSymptomHistory ? (
-        <GlobalText style={style.noSymptomHistoryText}>
-          {t("symptom_checker.no_symptom_history")}
+    <>
+      <View style={[style.headerContainer, last14DaysStyle]}>
+        <GlobalText style={style.headerText}>
+          {t("symptom_checker.last_14_days")}
         </GlobalText>
-      ) : (
-        dailyLogData.map((logData) => {
-          return <DaySummary key={logData.date} dayLogData={logData} />
-        })
-      )}
-    </ScrollView>
+        <TouchableOpacity>
+          <SvgXml
+            xml={Icons.Calendar}
+            width={Iconography.xSmall}
+            height={Iconography.xSmall}
+            testID="calendar-button"
+            onPress={() => {
+              setViewSelection(ViewSelection.CALENDAR)
+            }}
+          />
+        </TouchableOpacity>
+      </View>
+      <View style={[style.headerContainer, historyStyle]}>
+        <GlobalText style={style.headerText}>
+          {t("symptom_checker.history")}
+        </GlobalText>
+        <TouchableOpacity>
+          <SvgXml
+            xml={Icons.Hamburger}
```

Comment:
Made a note below about using the `fill` prop here. This way we can change this fill color easily without having to dig around in the svg.

```
<SvgXml
  fill={Colors.primary100}
  ...
/>
```